src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -653,6 +713,7 @@ public void Serialize_RoundtripWithUnownedHandle_Success()
         }
 
         [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "The hardcoded bytes depend on the existence of the System.Drawing.Common assembly.")]
```

Comment:
No, we should remove [Serializable] from all the drawing types since we don't plan to support it. I thought there was a tracking issue but I can't find it. Of course, with care wrt Mono.